### PR TITLE
Bug 1108632 Implement Firefox OS CES landing page

### PR DESCRIPTION
**Pull request by sgarrity**
A relative basic static page using the mozilla-pager-js to scroll through multiple screenshots.

To launch before Jan 5.

### CODE REVIEW

**Comment by sgarrity** [comment]
I've made a variety of tweaks based on feedback in the bug. They are in separate commits for easier review, but I'll squash once I get an updated r+ (@agibson?)

This is up on https://www-demo1.allizom.org/en-US/firefox/os/devices/tv/

File: bedrock/firefox/templates/firefox/os/tv.html [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+</div>
+
+<div id="main-content">
+
+  <div class="pager pager-auto-init pager-with-nav pager-with-tabs pager-no-history" id="mozilla-pager-1">
+
+    <div class="tv">
+      <div class="pager-content" id="mozilla-pager-tv-pages">
+        <div id="tv1" class="pager-page">
+          {{ high_res_img('img/firefox/os/devices/tv/screen1.jpg', {'alt': _('Screenshot'), 'width': '796', 'height': '448'}) }}
```
Comment (craigcook) [inline bedrock/firefox/templates/firefox/os/tv.html]: This alt text isn't especially useful. If we can't describe what's on the screen we should just have empty alt.

File: bedrock/firefox/templates/firefox/os/tv.html [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+        </div>
+        <div id="tv4" class="pager-page">
+          {{ high_res_img('img/firefox/os/devices/tv/screen4.jpg', {'alt': _('Screenshot'), 'width': '796', 'height': '448'}) }}
+        </div>
+      </div>
+    </div>
+
+    <ol class="pager-tabs">
+      <li>
+        <a href="#tv1"><img src="{{ media('img/firefox/os/devices/tv/screen1.jpg') }}" alt="{{ _('Screenshot') }}"></a>
```
Comment (craigcook) [inline bedrock/firefox/templates/firefox/os/tv.html]: Here the alt text is also the link label, so there's nothing distinguishing these four links from each other. At the very least this should be something like "View the first screenshot" (and so on for the other three). Even better, "View a screenshot of ___" if we can rustle up some short labels for what each one is.

Comment (sgarrity) [inline bedrock/firefox/templates/firefox/os/tv.html]: I'm not sure "View the first screenshot" is any better than "View screenshot" if we don't have meaningful descriptions of the screenshots either.

I agree that these ALTs aren't great, but I opted to be generic as we're under a string freeze for l10n.

Comment (craigcook) [inline bedrock/firefox/templates/firefox/os/tv.html]: Since we're already string-frozen I guess this will have to do. Not a blocker anyway.

**Comment by craigcook** [comment]
Looks good to me! r+ :tv: 
Give it a squash and we're set to merge.

**Pull request merged**